src/lightning_lite/utilities/__init__.py

```diff
@@ -0,0 +1,42 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""General utilities."""
+
+from lightning_lite.utilities.apply_func import move_data_to_device  # noqa: F401
+from lightning_lite.utilities.distributed import AllGatherGrad  # noqa: F401
+from lightning_lite.utilities.enums import _AcceleratorType, _StrategyType, AMPType, LightningEnum  # noqa: F401
+from lightning_lite.utilities.imports import (  # noqa: F401
```

Comment:
can we use this to directly move the availability checks to the respective modules (in a follow-up maybe)?

Reply:
Definitely would like that. Right now, the modules are not there yet, so it has to come in a follow up. This isn't high priority now, but once all major parts have moved we will definitely have the freedom to move these things around within Lite.